tests/test_agent.py

```diff
@@ -100,25 +102,20 @@ def test_run_agent():
                 error_msg="Agent loop not started!'",
             )
             wait_for_condition(
-                lambda: agent.agent_state == AgentState.RUNNING,
+                lambda: agent.state == RuntimeStates.running,
                 timeout=5,
                 error_msg="Agent state must be 'running'",
             )
+            wait_for_condition(
+                lambda: agent.agent_state == AgentState.RUNNING,
```

Comment:
why do we have both `agent_state` and `state`?

Reply:
it was my first comment in this pr about what to do with this agent_state.

now it removed.